src/app/login/login.component.html

```diff
@@ -11,21 +11,21 @@
     </div><!--/.col-*-->
 
     <div class="col-sm-7 col-md-6 col-lg-5 login">
-      <form class="form-horizontal" role="form">
+      <form id="githubLogin" class="form-horizontal" role="form">
```

Comment:
@sanbornsen I think this comment is referring to an older commit. It does not reflect the changes. Regardless you are correct, the IDs should refer to the "thing" they point to.